pkg/controller/worker/machines_test.go

```diff
@@ -208,7 +212,11 @@ var _ = Describe("Machines", func() {
 				zone1 = region + "a"
 				zone2 = region + "b"
 
+				AwsCsiDriverTopologyKey = "topology.ebs.csi.aws.com/zone"
```

Comment:
Can you reuse [this const](https://github.com/gardener/gardener-extension-provider-aws/pull/365/files#diff-b34b29ff5c37b47738c2ac7ef24c03eaca1a5c54e7635201747422c924661a32R32) here?